pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/ProxyStats.java

```diff
@@ -33,13 +30,13 @@
      * @throws PulsarAdminException
      */
 
-    JsonArray getConnections() throws PulsarAdminException;
```

Comment:
Since we have discussed in the community meeting, here need to introduce breaking change to keep the API module simple, I agree with using string the returned type, it would be better if we can add a description to indicate the return string in a JSON format.